src/cmd-run

```diff
@@ -259,6 +259,14 @@ case "${DISK_CHANNEL}" in
     *) die "Invalid --disk-channel ${DISK_CHANNEL}" ;;
 esac
 
+if [ "${IMAGE_TYPE}" == metal4k ]; then
+    kola_args+=("--qemu-native-4k")
+    # native 4k requires a UEFI bootloader
```

Comment:
Hum.  Possibly we should drop the BIOS partition then in this case?

Reply:
I was trying to keep it close to `metal`, but hmm yeah I think that makes sense.